spec/factory_bot/find_definitions_spec.rb

```diff
@@ -109,4 +109,36 @@ def self.in_directory_with_files(*files)
       end
     end
   end
+
+  describe "definition_file_paths" do
+    in_directory_with_files "spec/my_factories/factory_one.rb", "spec/my_factories/factory_two.rb"
+
+    before { allow(FactoryBot).to receive(:load) }
+
+    it "supports files without extension" do
```

Comment:
We've already testing supporting files without extension and directories, so I don't think we need additional coverage there. I think the last test is enough.

Reply:
I think all the existing coverage is implicit about the contents of `definition_file_paths`, this was meant to pin the supported cases.

I can surely remove the two extra examples, is that your preference?

Reply:
Yeah, I see what you mean. It looks like the only place we actually test setting `definition_file_paths` is in a cucumber test.

Let's keep your additional test, but instead of stubbing `definition_file_paths` set it with something like:

```rb
it "" do
  old_paths = FactoryBot.definition_file_paths
  FactoryBot.definition_file_paths = ["whatever/paths/you/want/for/that/test"]
  
  # test stuff
ensure
  FactoryBot.definition_file_paths = old_paths
end
```

That will point us in the direction of eventually getting rid of the cucumber specs.